src/webpack/loaders/devSitePropsTable.js

```diff
@@ -1,59 +1,93 @@
 const { getOptions } = require('loader-utils');
-const mdx = require('@mdx-js/mdx');
 const reactDocs = require('react-docgen');
 const findSource = require('../loaderUtils/findSource');
 
 /**
- * Create an inline mdx react component.
- * @param {*} description
- * @param {*} callback
+ * Parses prop types into a more compact representation.
+ * @param {object} type prop type to parse.
```

Comment:
I like to call out the specific elements in the object if possible: 

```suggestion
 * @param {object} type prop type information to parse.
 * @param {string} type.value the value for the type
 * @param {string} type.name the name of the type
 * @param {boolean} type.required whether or not the type is required
 * @param {description} type.description the description of the type
```